WebViewControl.Avalonia/WebView.InternalDragHandler.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xilium.CefGlue;
+using Xilium.CefGlue.Common.Handlers;
+
+namespace WebViewControl {
+    partial class WebView {
```

Comment:
You have 2 implementations of the InternalDragHandler ... you should have one in the WebViewControl and link in the avalonia variant